web/src/views/FindSpeaker.vue

```diff
@@ -96,19 +124,47 @@ export default {
     setError(err) {
       this.error = err;
     },
+    getPreviousPage() {
+      if (this.page > 1) {
+        this.page -= 1;
+        this.isMaxPage = false;
+      }
+    },
+    getNextPage() {
+      // leave empty, this function is overwritten by the getSpeakers method
```

Comment:
why?

Reply:
same here, can we just embrace the pagination and fetch the next page when the user clicks the next page?

Reply:
I explained it a bit more in detail here:
https://github.com/WWCodeTokyo/speak-her-db/pull/166#issuecomment-703031161

one option would be to have a `getNextPageInternal` function with our actual pagination logic which is currently sitting in the `done` func we pass into `eachPage`:
https://github.com/WWCodeTokyo/speak-her-db/pull/166/files#diff-8d57168f5b367bbc58eeb3b2b3950debR160-R164

and instead of overwriting `getNextPage` we could set a different property on the component with the `next` function returned by airtablejs and have `getNextPage` handle the logic for which one to call.

kind of like this:
```javascript
      // ...
     airTableNext() {
        // placeholder, this will be overwritten by the next method returned by airtablejs
       // will be set to `undefined` when the done function passed into eachPage is called which marks the last page was reached.
     },
     getNextPageInternal() {
       if (!this.isMaxPage) {
         this.page += 1;
         const lastPageEntry = this.page * this.pageSize;
         this.isMaxPage = lastPageEntry >= this.speakers.length;
       }
     },
     getNextPage() {
         if (this.airTableNext) {
             this.airTableNext();
             return;
         }
         this.getNextPageInternal();
     }
```

Reply:
I implemented it this way for now, I think it's a bit more readable / easier to follow the logic? :)

https://github.com/WWCodeTokyo/speak-her-db/pull/166/commits/69d4937b1db5ba70d30a45c4485fc21a20f8a7a3